pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2206,6 +2206,12 @@ private void updateBrokerDispatchThrottlingMaxRate() {
     }
 
     private void updateBrokerPublisherThrottlingMaxRate() {
+        forEachTopic(topic -> {
```

Comment:
This is not right entrance.
It seems that we missed triggering the update of `publishRate` when broker level settings updated.